crates/stc_ts_type_checker/tests/conformance/controlFlow/controlFlowDeleteOperator.ts.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
-    required_error: 1,
-    matched_error: 1,
-    extra_error: 0,
+    required_error: 0,
+    matched_error: 2,
+    extra_error: 1,
```

Comment:
is it possible to get the type of a member and only that type:

```ts
var a: { k: { d: string } } | undefined

delete a.k.d

```

when accessing, stc type_of_member_expr currently returns string | undefined, instead of just string

Reply:
I think this is a bug of `access_property`.
IIRC it should return `string` or return an error